docs/spec/api.md

```diff
@@ -1,7 +1,5 @@
 # Docker Registry HTTP API V2
 
-## Introduction
```

Comment:
This edit needs to be made to the template and then regenerated with 'make docs/spec/api.md`. That target may be to be updated to find this.